syft/format/common/cyclonedxhelpers/to_format_model.go

```diff
@@ -44,6 +75,33 @@ func ToFormatModel(s sbom.SBOM) *cyclonedx.BOM {
 	return cdxBOM
 }
 
+func digestsToHashes(digests []file.Digest) *[]cyclonedx.Hash {
+	hashes := make([]cyclonedx.Hash, len(digests))
+	for i, digest := range digests {
+		cdxAlgo := toCycloneDXAlgorithm(digest.Algorithm)
```

Comment:
Do we need to handle a map miss here? What should happen if there's a digest for an algorithm outside CDX spec? For example, what if someone crafts an SBOM with xx64 hashes of files? (I'm not sure any current spec allows this, but I think right now we would end up with a blank algorithm field, which is not allowed: https://cyclonedx.org/docs/1.6/json/#components_items_hashes_items_alg

Reply:
Yep! That's a good point. I was only thinking about syft generated SBOMs who's digest algorithm are a subset of the current cdx specification.